components/transfer/Transfer.vue

```diff
@@ -1,215 +1,274 @@
+          </a>
+        </div>
+        <Auth v-else />
+      </div>
+      <div class="is-flex is-flex-direction-column is-align-items-end">
+        <span class="has-text-weight-bold is-size-6 mb-1">{{
+          $t('general.balance')
+        }}</span>
+        <div class="is-flex is-align-items-center">
+          <img class="mr-2 is-32x32" :src="tokenIcon" alt="token" />
+          <Money :value="balance" inline />
+        </div>
+
+        <span class="has-text-grey">≈ ${{ balanceUsdValue }}</span>
+      </div>
+    </div>
+
+    <hr />
+
+    <div class="is-flex">
+      <div class="is-flex-grow-1 mr-2 is-flex is-flex-direction-column">
+        <div class="has-text-weight-bold is-size-6 mb-3">
+          {{ $t('transfers.recipient') }}
+        </div>
+        <div
+          v-for="(destinationAddress, index) in targetAddresses"
+          :key="index"
+          class="mb-3">
+          <AddressInput
+            v-model="destinationAddress.address"
+            label=""
+            placeholder="Enter wallet address"
+            :strict="false" />
+        </div>
       </div>
 
-      <DisabledInput
-        v-show="
-          correctAddress(destinationAddress) &&
-          correctAddress(destinationAddress) !== destinationAddress
-        "
-        :label="$t('general.correctAddress')"
-        :value="correctAddress(destinationAddress)" />
+      <div class="is-flex is-flex-direction-column">
+        <div class="has-text-weight-bold is-size-6 mb-3">
+          {{ $t('amount') }}
+        </div>
+        <div
+          v-for="(destinationAddress, index) in targetAddresses"
+          :key="index"
+          class="mb-3">
+          <NeoInput
+            v-if="displayUnit === 'token'"
+            v-model="destinationAddress.token"
+            type="number"
+            placeholder="0"
+            step="0.01"
+            min="0"
+            icon-right-class="search"
+            @input="onAmountFieldChange(destinationAddress)" />
+          <NeoInput
+            v-else
+            v-model="destinationAddress.usd"
+            placeholder="0"
+            type="number"
+            step="0.01"
+            min="0"
+            icon-right-class="search"
+            @input="onUsdFieldChange(destinationAddress)" />
+        </div>
+      </div>
     </div>
-    <div class="container mb-3">
-      <NeoField>
-        <BalanceInput
-          v-model="price"
-          :label="$t('amount')"
-          :calculate="false"
-          @input="onAmountFieldChange" />
-      </NeoField>
-      <NeoField>
-        <ReadOnlyBalanceInput
-          v-model="usdValue"
-          :label-input="$t('teleport.usdInput')"
-          label="USD"
-          @input="onUSDFieldChange" />
-      </NeoField>
+
+    <div
+      class="mb-5 is-flex is-justify-content-center is-clickable"
+      @click="addAddress">
+      {{ $t('transfers.addAddress') }}
+      <NeoIcon class="ml-2" icon="plus" pack="fas" />
+    </div>
```

Comment:
how about NeoButton?

Reply:
There is no available variant of NeoButton for this design. I will keep this simple implementation.